frontend/src/router/dashboard/projectV1.ts

```diff
@@ -88,16 +88,16 @@ const projectV1Routes: RouteRecordRaw[] = [
         children: [
           {
             path: "",
-            name: PROJECT_V1_ROUTE_MASKING_ACCESS,
+            name: PROJECT_V1_ROUTE_MASKING_EXEMPTION,
```

Comment:
Do we need to update the route too? `masking-access` -> `masking-exemption`

https://github.com/bytebase/bytebase/blob/main/frontend/src/router/dashboard/projectV1.ts#L78

Reply:
Thanks for catching it! Done.